smt/surrogate_models/krg_based.py

```diff
@@ -220,7 +220,7 @@ def _final_initialize(self):
 
         # initialize default power values
         if self.options["corr"] == "squar_exp":
-            self.options["pow_exp_power"] = 2.0
+            self.options["pow_exp_power"] = 2.0 - 1e-8
```

Comment:
Hmmm... Sorry but **squared** exponential exponent is equal to 2!  ;-) I guess you try to handle a numerical issue but it cannot be done like that.